packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -130,6 +130,8 @@ describe('Popover', () => {
     /**
      * FIXME: some of these tests, including style snapshots, throw act()
      * warnings. We should look into it.
+     *
+     * NOTE: the offset behavior enabled by the `offset` prop is tested with Chromatic via storybook
```

Comment:
We could mention that we're relying on Chromatic because jsdom doesn't support this. Here's a suggestion, but feel free to rephrase/add details 🙂 

```suggestion
     * NOTE: we can't test the offset behavior enabled by the `offset`
     * prop with `jsdom`. The logic is covered in Chromatic.
```

Reply:
Makes sense! I've committed the suggested change :)